### PR TITLE
Add Slack channel for kustomize training cohort

### PR DESCRIPTION
**Pull request by annasong20**
Create Slack channel that will be archived once kustomize maintainer training cohort finishes.

Reference #7401 

/cc @natasha41575

### CODE REVIEW

File: communication/slack-config/channels.yaml [hunk 1]
```diff
@@ -291,6 +291,7 @@ channels:
   - name: kudo
   - name: kurl
   - name: kustomize
+  - name: kustomize-training-cohort # archive once cohort finishes
```
Comment (natasha41575) [inline communication/slack-config/channels.yaml:294]: Is there some configuration we need to add here to make this a private channel? 

cc @mrbobbytables

Comment (annasong20) [reply communication/slack-config/channels.yaml:294]: Would love @mrbobbytables feedback.

I wrote this based on previous PRs I saw. I know that we can also specify members?

**Comment by natasha41575** [comment]
Hey @annasong20 sorry for the confusion/miscommunication, talked with Bob and I think this PR is not needed. The slack admins will handle creation of the channel for the cohort.

**Pull request closed without merging**